scripts/run_linter.sh

```diff
@@ -68,13 +68,13 @@ if [ ! -d "$TOOLS_DIR/pycodestyle-2.3.1" ]; then
 fi
 
 if [ "$TRAVIS" == 'true' ]; then
-  pycodestyle -v --exclude=./node_modules,.git,./.vscode,./.circleci || exit 1
-  pydocstyle -v || exit 1
+  pydocstyle --match-dir="^(?!oppia_tools|\.vscode).*" -v || exit 1
```

Comment:
Perhaps add some coments to explain the arg values here -- I'm not sure it's self-evident.

Reply:
Done :)